app/src/main/AndroidManifest.xml

```diff
@@ -15,6 +15,7 @@
         android:roundIcon="@mipmap/ic_launcher_antivirus_round"
         android:supportsRtl="true"
         android:theme="@style/AppTheme">
+        <activity android:name=".features.auth.SignInActivity"></activity>
```

Comment:
`portrait` and `/>` ?

Reply:
I suppose that CW supports vertical screen.

Reply:
Vertical (portrait) only for simplicity matters.